src/test/java/com/thealgorithms/strings/AnagramsTest.java

```diff
@@ -1,23 +1,18 @@
 package com.thealgorithms.strings;
 
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
 
 public class AnagramsTest {
-
-    @Test
-    public void isAlphabetical() {
-        String input1 = "late";
-        Anagrams anagrams = new Anagrams();
-        assertTrue(anagrams.approach1(input1, "tale"));
-        assertTrue(anagrams.approach1(input1, "teal"));
-        assertTrue(anagrams.approach2(input1, "tale"));
-        assertTrue(anagrams.approach2(input1, "teal"));
-        assertTrue(anagrams.approach3(input1, "tale"));
-        assertTrue(anagrams.approach3(input1, "teal"));
-        assertTrue(anagrams.approach4(input1, "tale"));
-        assertTrue(anagrams.approach4(input1, "teal"));
-        assertTrue(anagrams.approach5(input1, "teal"));
+    @ParameterizedTest
+    @CsvSource({"late, tale, true", "late, teal, true", "listen, silent, true", "hello, olelh, true", "hello, world, false", "deal, lead, true", "binary, brainy, true", "adobe, abode, true", "cat, act, true", "cat, cut, false"})
+    void testApproach1(String input1, String input2, boolean expected) {
```

Comment:
Could you add separate tests for each approach that will all use the same test case source?

Reply:
Sure. Fixed.

Reply:
Thanks.